seq2science/rules/get_fastq.smk

```diff
@@ -63,11 +63,12 @@ rule run2sra:
 
 
 # ENA > SRA
-ruleorder: ena2fastq_SE > sra2fastq_SE
-ruleorder: ena2fastq_PE > sra2fastq_PE
+ruleorder: ena2fastq_SE > gsa2fastq_SE > sra2fastq_SE
+ruleorder: ena2fastq_PE > gsa2fastq_PE > sra2fastq_PE
```

Comment:
i assume gsa is used least, so maybe put it last?

Reply:
doesn't really matter, they are mutually exclusive